.ahub/sam/exclude.txt

```diff
@@ -1,53 +1,49 @@
 # External code: Android NN API
-/ONE/compiler/ann-api/include/NeuralNetworks.h
```

Comment:
This works for /nnfw/XXX ?

Reply:
I checked this updates for ONE only and it worked.